src/contact-author.ts

```diff
@@ -0,0 +1,75 @@
+import * as sourcegraph from 'sourcegraph'
+import { EMPTY, from, Observable } from 'rxjs'
+import { filter, map, switchMap } from 'rxjs/operators'
+import { Hunk, queryBlameHunks, resolveURI } from './blame'
+
+const decorationType = sourcegraph.app.createDecorationType()
+
+function observeActiveCodeEditorChanges(): Observable<sourcegraph.CodeEditor> {
```

Comment:
This is a really nice way to clean up the activate function! We should expose an API like this since it looks like we use this so often.